src/ip/QosConfig.cc

```diff
@@ -215,7 +277,7 @@ Ip::Qos::Config::Config() : tosLocalHit(0), tosSiblingHit(0), tosParentHit(0),
     markParentHit(0), markMiss(0), markMissMask(0),
     preserveMissMark(false), preserveMissMarkMask(0xFFFFFFFF),
     tosToServer(NULL), tosToClient(NULL), nfmarkToServer(NULL),
-    nfmarkToClient(NULL)
+    nfmarkToClient(NULL), nfConnmarkToClient(NULL)
```

Comment:
nullptr in new and changed code please. Although if you make the new nfConnmarkToClient definition in the header use C++11 initialization this change becomes unnecessary. Cleanup of the rest of this class can be left out of scope, the two types of initialization work fine when mixed.